blazar/plugins/floatingips/floatingip_plugin.py

```diff
@@ -370,3 +389,34 @@ def delete_floatingip(self, fip_id):
         except db_ex.BlazarDBException as e:
             raise manager_ex.CantDeleteFloatingIP(floatingip=fip_id,
                                                   msg=str(e))
+
+    def query_allocations(self, resource_id_list, lease_id=None,
+                          reservation_id=None):
+        return self.query_fip_allocations(resource_id_list, lease_id=lease_id,
+                                          reservation_id=reservation_id)
+
+    def query_fip_allocations(self, fips, lease_id=None, reservation_id=None):
+        """Return dict of host and its allocations.
+
+        The list element forms
+        {
+          '-id': [
+                       {
+                         'lease_id': lease_id,
+                         'id': reservation_id
+                       },
+                     ]
+        }.
+        """
+        start = datetime.datetime.utcnow()
+        end = datetime.date.max
+
+        # To reduce overhead, this method only executes one query
```

Comment:
I don't quite get this comment - what is the alternative?

Reply:
This was a comment left over from a copy and paste from query_host_allocations to implement query_fip_allocations. I believe it's referring to the query in the db layer.